fastlane/lib/fastlane/actions/unlock_keychain.rb

```diff
@@ -82,12 +82,12 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :add_to_search_list,
                                        env_name: "FL_UNLOCK_KEYCHAIN_ADD_TO_SEARCH_LIST",
                                        description: "Add to keychain search list",
```

Comment:
Should we add that that the valid values are `true`, `false`, `:add`, and `:replace` I the description?